aries-site/src/examples/components/card/CardAnatomy.js

```diff
@@ -6,7 +6,7 @@ import { Info, Emoji } from 'grommet-icons';
 const spacing = defaultProps.theme.global.edgeSize;
 
 export const CardAnatomy = () => (
-  <Card width="medium">
+  <Card background="background" width="medium">
```

Comment:
I missed this in last week's Card PR. I think we have opportunities to set some default backgrounds and padding in the theme. I have those defined and will submit in a separate PR along with updates to these examples for cleanup purposes.

For the scope of this PR, can we change this to:
```suggestion
  <Card background="background-front" width="medium">
```

This will help with visibility on Dark mode.